docs/data-sources/directory.md

```diff
@@ -37,33 +37,36 @@ data "btp_directory" "by_id" {
-- `last_modified` (String) The date and time the resource was last modified in [RFC3339](https://www.ietf.org/rfc/rfc3339.txt) format.
+- `last_modified` (String) The date and time when the resource was last modified in [RFC3339](https://www.ietf.org/rfc/rfc3339.txt) format.
 - `name` (String) The display name of the directory.
 - `parent_id` (String) The GUID of the directory's parent entity. Typically this is the global account.
 - `state` (String) The current state of the directory. Possible values are: 
-	 - `OK` The CRUD operation or series of operations completed successfully.
-	 - `STARTED` CRUD operation on an entity has started.
-	 - `CREATING` Creating entity operation is in progress.
-	 - `UPDATING` Updating entity operation is in progress.
-	 - `MOVING` Moving entity operation is in progress.
-	 - `PROCESSING` A series of operations related to the entity is in progress.
-	 - `DELETING` Deleting entity operation is in progress.
-	 - `PENDING REVIEW` The processing operation has been stopped for reviewing and can be restarted by the operator.
-	 - `CANCELLED` The operation or processing was canceled by the operator.
-	 - `CREATION_FAILED` The creation operation failed, and the entity was not created or was created but cannot be used.
-	 - `UPDATE_FAILED` The update operation failed, and the entity was not updated.
-	 - `PROCESSING_FAILED` The processing operations failed.
-	 - `DELETION_FAILED` The delete operation failed, and the entity was not deleted.
-	 - `MOVE_FAILED` Entity could not be moved to a different location.
-	 - `MIGRATING` Migrating entity from NEO to CF.
+
+  | state | description |
+  | ----- | ----------- |
+  | OK | The CRUD operation or series of operations completed successfully. |
+  | STARTED | CRUD operation on an entity has started. |
+  | CANCELLED | The operation or processing was canceled by the operator. |
+  | PROCESSING | A series of operations related to the entity is in progress. |
+  | PROCESSING_FAILED | The processing operations failed. |
+  | CREATING | Creating entity operation is in progress. |
+  | CREATION_FAILED | The creation operation failed, and the entity was not created or was created but cannot be used. |
+  | UPDATING | Updating entity operation is in progress. |
+  | UPDATE_FAILED | The update operation failed, and the entity was not updated. |
+  | MOVING | Moving entity operation is in progress. |
+  | MOVE_FAILED | Entity could not be moved to a different location. |
+  | DELETING | Deleting entity operation is in progress. |
+  | DELETION_FAILED | The delete operation failed, and the entity was not deleted. |
+  | PENDING REVIEW | The processing operation has been stopped for reviewing and can be restarted by the operator. |
+  | MIGRATING | Migrating entity from NEO to CF. |
 - `subdomain` (String) Applies only to directories that have the user authorization management feature enabled. The subdomain is part of the path used to access the authorization tenant of the directory.
```

Comment:
This is how it looks in the https://registry.terraform.io/tools/doc-preview:
![Screenshot 2023-06-23 at 16 19 36](https://github.com/SAP/terraform-provider-btp/assets/136796355/0e30b960-aed1-4520-976d-669f0e0a21c1)